lib/src/pubspec.dart

```diff
@@ -112,6 +112,21 @@ class Pubspec {
 
   SdkConstraintStatus get sdkConstraintStatus =>
       SdkConstraintStatus.fromSdkVersion(_inner.environment['sdk']);
+
+  /// The sdk tags associated with [this].
+  List<String> sdkTags() {
+    final result = <String>[];
+    final dependentSdks = this.dependentSdks;
+    if (dependentSdks.every(['dart'].contains)) {
```

Comment:
I find this hard to parse. Isn't this the same as:
```dart
if (dependentSdks.length == 1 && dependentSdks.single =='dart') {
```

Reply:
Yes, I wrote it like that initially, but changed it to follow the same pattern as below.